docs/extend/extend-zowe-overview.md

```diff
@@ -1,56 +1,60 @@
+Service registration of a REST API to the API ML is performed through a call to the Discovery Service component of API ML which contains registration data and metadata for that service. Registration requires the z/OS service to know the web address of the API Discovery Service. When Dynamic registration is performed, the service that performs the registration must periodically send heartbeat requests for each registered service instance to the Discovery Service. These requests enable the Discovery Service to monitor the availability of registered service instances. Services that are registered dynamically display the status of the service in the API Catalog after initial service registration. Services that are registered statically also appear in the API Catalog, but the status of service instances is not displayed. 
 
-The Zowe z/OS started task `ZWESVSTC` that launches the Zowe address spaces allows for extra USS 'microservices' to be lifecycled with it, so that they are started together with Zowe and ended when Zowe started task is stopped. For more information, see [Lifecycling with Zowe](lifecycling-with-zwesvstc.md).  This can be used, for example, to start and stop a dynamic APIML Spring Boot micro service that provides its own APIs or acts as a broker to register APIs on behalf of an existing z/OS server.   
+For more information about how to build a service which is able to register, see the [Onboarding Overview](extend-apiml/onboard-overview.md)  
 
 ### Static API registration
 
-Instead of having the API service calling up to the API Mediation Layer, it is possible to tell the API Mediation Layer about an API service by giving it a static file with details of the z/OS API service.  This is referred to in the documentation as being able to onboard without code changes, because there is no need to modify the existing API service to have it call up to the API Mediation Layer, or introduce a Spring Boot micro service to do this on its behalf. For more information, see [Onboard a REST API without code changes required](extend-apiml/onboard-static-definition.md).
+For services that cannot be modified to make them dynamically discoverable, it is possible onboard to the API ML by providing a static file with details of the z/OS API service to the API ML. Zowe documentation refers to this form of registration as onboarding without code changes. This registration method does not require modifications to the existing API service for the service to call the API Mediation Layer. For more information, see [Onboard a REST API without code changes required](extend-apiml/onboard-static-definition.md). The status of services onboarded to the API ML through static registration, however, is not displayed in the API Catalog.
 
```

Comment:
For services that cannot be modified to be dynamically discoverable, it is possible onboard them to the API ML by providing to the API ML a static definition file with API service details. This registration method does not require modifications to the existing API service code. For more information, see [Onboard a REST API without code changes required](extend-apiml/onboard-static-definition.md). The availability status of services onboarded to the API ML through static registration, however, is not displayed in the API Catalog.